docs/_static/custom.css

```
@@ -1,395 +1,738 @@
-    margin-bottom: 8px;
-    display: flex;
-    align-items: center;
-}
-/* Add logo icon before title if possible via CSS? Hard in pure CSS for sphinx admonition title */
-.admonition .admonition-title::before {
-    content: "💡"; /* Fallback icon */
-    margin-right: 8px;
-    color: var(--orion-blue);
+  background-color: transparent !important;
+  color: var(--orion-blue) !important;
+  font-family: "Satoshi", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Satoshi" (font-family-name-quotes)

(font-family-name-quotes)